docs/podman-create.1.md

```diff
@@ -659,8 +659,7 @@ Note: Labeling can be disabled for all containers by setting label=false in the
 
 **--shm-size**=*size*
 
-Size of `/dev/shm`. The format is `<number><unit>`. `number` must be greater than `0`.
```

Comment:
I'll let @mheon correct me, but I think we still need the "> zero" warnings still for the --shm-size option.

Reply:
According to [tmpfs kernel doc](https://www.kernel.org/doc/Documentation/filesystems/tmpfs.txt), `size=0` can be used to specify an unlimited amount of memory, so `--shm-size=0` is something legit.

Reply:
Does the tool currently check this?  Does it handle it?

Reply:
When you pass `--shm-size=0`, the `size=0` option is passed to `mount(2)`, hence it is handled and no special support is required. 
In case, I can add  the `--shm-size=0` use case in the man page.

Reply:
@marcov Once this is resolved, you are mergable.

Reply:
I added the following disclaimer in the man pages:
```
When size is `0`, there is no limit on the amount of memory used for IPC by the container.
```

If you really want to disallow a size of `0`, then podman should check that, saying *must be greater than `0`* is IMO wrong.